core/docker/build.sh

```diff
@@ -66,18 +71,38 @@ tar -C "${WORK_DIR}" -xzf "${WORK_DIR}/trino-server-${TRINO_VERSION}.tar.gz"
 rm "${WORK_DIR}/trino-server-${TRINO_VERSION}.tar.gz"
 cp -R bin "${WORK_DIR}/trino-server-${TRINO_VERSION}"
 cp -R default "${WORK_DIR}/"
+BASE_WORK_DIR="$(mktemp -d)"
 
+BASE_PREFIX="trino-base:${TRINO_VERSION}"
 TAG_PREFIX="trino:${TRINO_VERSION}"
 
+echo "👷 Docker buildx builders available:"
+docker buildx ls
+
 for arch in "${ARCHITECTURES[@]}"; do
-    echo "🫙  Building the image for $arch"
-    docker build \
-        "${WORK_DIR}" \
-        --pull \
-        --platform "linux/$arch" \
-        -f Dockerfile \
-        -t "${TAG_PREFIX}-$arch" \
-        --build-arg "TRINO_VERSION=${TRINO_VERSION}"
+    echo "🫙  Building the base image for $arch"
+    BASE_BUILD_OPTIONS=(
+        ${BASE_WORK_DIR}
```

Comment:
Is this only to have an empty context? Maybe add a comment.

Reply:
Yes, it minimizes the build context which we don't need in the base image